devtools/aesh/src/main/java/io/quarkus/cli/commands/ListExtensionsCommand.java

```diff
@@ -23,10 +23,10 @@
     @Option(shortName = 'h', hasValue = false)
     private boolean help;
 
-    @Option(shortName = 'a', hasValue = false, description = "Display name, group-id, artifact-id and version (default behaviour)")
+    @Option(shortName = 'a', hasValue = false, description = "Display all extensions or just the installable.")
```

Comment:
what about _compatible_ ones?

Reply:
In fact we have no notion of compatibility, we just check if this already installed or not

Reply:
In fact we have no notion of compatibility, we just check if this already installed or not